handler.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var craigslistApartments = require('./craigslist-apartments');
+var filters = require('./filters');
+const Filter = new filters();
+
+module.exports.hello =   (event, context) => {
+  var apartmentsPromise = craigslistApartments.getListOfRentals('0',[]);
```

Comment:
Before getting the list of rentals, you should create the filter object from the lambda input (check https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-handler.html for examples).